package.json

```diff
@@ -63,12 +63,14 @@
   "dependencies": {
     "@types/bn.js": "^4.11.6",
     "@types/elliptic": "^6.4.12",
+    "@types/lodash": "^4.14.149",
     "@types/randombytes": "^2.0.0",
     "@types/ripemd160": "^2.0.0",
     "@types/sha.js": "^2.4.0",
     "bn.js": "^4.11.8",
     "c32check": "^1.0.1",
     "elliptic": "^6.5.2",
+    "lodash": "^4.17.15",
```

Comment:
May want to run `npm audit fix`. Lots of security warnings after an `npm i` on this branch